src/lightning_app/cli/lightning_cli.py

```diff
@@ -116,6 +121,51 @@ def run_app(
     _run_app(file, cloud, without_server, no_cache, name, blocking, open_ui, env)
 
 
+@run.command("command")
+@click.argument("command", type=str, default="")
+@click.option(
+    "--args",
+    type=str,
+    default=[],
+    multiple=True,
+    help="Arguments to be passed to the method executed in the running app.",
+)
+@click.option(
+    "--id", help="Unique identifier for the application. It can be its ID, its url or its name.", default=None, type=str
```

Comment:
It can't be id unless we have id also for the regular run app command. For now we should just use name. We have not yet formally introduced what an ID is for an app, but users know name and it's always unique for a single user.

Reply:
The problem with the name, it is a bit limited.  Ideally, you would like to trigger a method on the app of someone else.

Reply:
Added support for some validation layers, especially with ClientCommand.

Reply:
do you think it's better to rename it to `app-id` or `app_id`.

From a security/ authorization perspective, how can an app developer restrict running commands from anyone who has either username/app-name or app_id (which are public info)?